features/reloading.feature

```diff
@@ -63,3 +63,45 @@ Feature:
     And I run `spring stop` with a clean environment
     Then the output should contain "1 runs, 1 assertions"
     And the output should not contain "Failure:"
+
+Scenario: Initializing the reloader with I18n support
```

Comment:
very self-explanatory and specific test 👏

Reply:
Thanks for the review! I will merge this and close #343 for now.